test/Microsoft.AspNetCore.Sockets.Tests/TestTaskExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Sockets.Tests
+{
+    public static class TestTaskExtensions
```

Comment:
Yes, which is not exactly unprecedented. I don't want any tests waiting indefinitely for things.